batch/scheduler/KProcessWrapper.class.php

```diff
@@ -121,8 +121,17 @@ public function getIndex()
 	 */
 	public function isRunning()
 	{
-		if($this->dieTime < time())
-			return false;
+		$usingSmartJobTimeout = ($this->taskConfig->type === 'KAsyncConvert' &&
```

Comment:
No need to check if type is KAsyncConvert , only those who have the flag will be checked